docs/content/en/docs/reference/nutanix/nutanix-prereq.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Requirements for EKS Anywhere on Nutanix Cloud Infrastructure"
+linkTitle: "Requirements"
+weight: 15
+description: >
+  Preparing a Nutanix Cloud Infrastructure provider for EKS Anywhere
+---
+
+To run EKS Anywhere, you will need:
+
+### Prepare Administrative machine
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+### Prepare a Nutanix environment
+To prepare a Nutanix environment to run EKS Anywhere, you need the following:
+* A Nutanix environment running AOS 5.20.4+ with AHV and Prism Central 2022.1+
+* Capacity to deploy 6-10 VMs
+* DHCP service or Nutanix IPAM running in your environment in the primary VM network for your workload cluster
+* One network in AOS to use for the cluster. EKS Anywhere clusters need access to Prism Central through the network to enable self-managing and storage capabilities.
+* A VM image imported into the Prism Image Service for the workload VMs
+* User credentials to create VMs and attach networks, etc
+* One IP address routable from cluster but excluded from DHCP/IPAM offering. 
+  This IP address is to be used as the [Control Plane Endpoint IP]({{< relref "../clusterspec/nutanix/#controlplaneconfigurationendpointhost-required" >}})
+
+  Below are some suggestions to ensure that this IP address is never handed out by your DHCP server. 
+ 
+  You may need to contact your network engineer.
+      
+   *  Pick an IP address reachable from cluster subnet which is excluded from DHCP range OR
+   *  Alter DHCP ranges to leave out an IP address(s) at the top and/or the bottom of the range OR
+   *  Create an IP reservation for this IP on your DHCP server. This is usually accomplished by adding 
+a dummy mapping of this IP address to a non-existent mac address.
+   *  Add an IP to the IPAM blacklist
+
+
+Each VM will require:
+
+* 2 vCPUs
+* 4GB RAM
+* 40GB Disk
+
+The administrative machine and the target workload environment will need network access to:
+
+{{% content "./domains.md" %}}
+
+
+### Nutanix information needed before creating the cluster
+You need to get the following information before creating the cluster:
+
+* **Static IP Addresses**: 
+You will need one IP address for the management cluster control plane endpoint, and a separate one for the controlplane of each workload cluster you add. 
+
+  Let’s say you are going to have the management cluster and two workload clusters.
+For those, you would need three IP addresses, one for each.
+All of those addresses will be configured the same way in the configuration file you will generate for each cluster.
+
+  A static IP address will be used for each control plane VM in your EKS Anywhere cluster.
```

Comment:
we only need 1 static IP for one cluster, not 1 for each control plane. I would re-phrase this
```suggestion
  A static IP address will be used for control plane API server HA in each of your EKS Anywhere cluster.
```